packages/mask/src/plugins/Collectible/helpers.ts

```diff
@@ -13,11 +13,11 @@ export const CollectibleMetadataReader = createTypedMessageMetadataReader<Collec
 )
 export const renderWithCollectibleMetadata = createRenderWithMetadata(CollectibleMetadataReader)
 
-export function toAsset(asset: { tokenId: string; tokenAddress: string; schemaName?: WyvernSchemaName }): Asset {
+export function toAsset(asset: { tokenId: string; tokenAddress: string; schemaName?: string }): Asset {
     return {
         tokenId: asset.tokenId,
         tokenAddress: asset.tokenAddress,
-        schemaName: asset.schemaName,
+        schemaName: asset.schemaName as WyvernSchemaName,
```

Comment:
i think need a assert function check the `schemaName` is correct.

see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions